src/components/Headers/NavBar.tsx

```diff
@@ -82,6 +85,8 @@ import getNFDsForAddress from '@/utils/accounts/getNFDsForAddress';
 import { useAsync } from 'react-use';
 import { useWallet } from '@txnlab/use-wallet';
 import { ellipseText, ellipseAddress } from '@/redux/helpers/utilities';
+import useSWR from 'swr';
+import { Dictionary } from '@reduxjs/toolkit';
```

Comment:
Make sure to fix this and all other eslint errors that you ci on failed ci pipeline

Reply:
ESlint errors fixed thanks to your help!